packages/client/tests/functional/extensions/model.ts

```diff
@@ -496,7 +496,7 @@ testMatrix.setupTestSuite(
       })
 
       expectTypeOf(args).toHaveProperty('include').toEqualTypeOf<null>()
-      expectTypeOf(args).toHaveProperty('where').toEqualTypeOf<{ id: '1' }>()
+      expectTypeOf(args).toHaveProperty('where').toMatchTypeOf({ id: '1' })
```

Comment:
Note: this is the only potentially "sketchy" change one might want to look at

Reply:
https://github.com/mmkal/expect-type
>To allow for extra properties, use .toMatchTypeOf. 

Did you see any extra properties?

Reply:
Some time ago I was looking at this problem. This PR is proposing to change the expected type from `{ id: '1' }` to `{ id: string }`. The test is passing, but is that correct to tweak the expected type?

I was looking if there is a similar test around. The one above is asserting:

https://github.com/prisma/prisma/blob/3a95e6ce0974a5df4ff5976eb11e79cef189e078/packages/client/tests/functional/extensions/model.ts#L470

Hm.. It can be that the assertion in question is correct and the test is failing, because of some change (or a bug) in newer versions TypeScript. The most visible difference between the failing and the passing tests is nullable type. I was trying to figure out what it could affect in Prisma types, but with no luck.

https://github.com/prisma/prisma/blob/3a95e6ce0974a5df4ff5976eb11e79cef189e078/packages/client/tests/functional/extensions/model.ts#L473-L476

Reply:
This could affect Pulse, definitely needs investigation.